src/components/users/MyAccountMenu/my-account-menu.scss

```diff
@@ -0,0 +1,63 @@
+@import "style/common/constants";
+@import "style/common/typography";
+
+.my-account-menu {
+  align-items: center;
+  display: flex;
+  height: 4vh;
+  flex-direction: column;
+  width: 15vw;
+}
+
+.border {
+  border-radius: 8px;
+}
+
+.border-top {
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+}
+
+.my-account-button {
+  align-content: center;
+  align-items: center;
+  background-color: $black-russian;
+  border-color: $white;
+  display: flex;
+  flex-direction: row;
+  height: 4vh;
```

Comment:
It is normally better to use padding rather than height or width since the button will support different text lengths and let the padding control the width and height depending on the text within the button.
Also it's a better practice to use rem as the css unit for this.

On the other hand, I'd recommend making components like buttons, inputs, headers, an other common elements as separate components with their own styles to avoid repeating code and save time